### DISCUSSION TITLE
Change license to MIT?

### DISCUSSION THREAD

**Discussion by DoctorFuchs** [DoctorFuchs/kiosk#48] [Polls]

**Comment by DoctorFuchs** [comment]
Please vote @codenius

**Comment by codenius** [comment]
Any reasons for license switch?

**Reply by DoctorFuchs** [reply]
Because MIT is more freely we give everyone the chance to use our software, without having the same license as we have.

**Reply by codenius** [reply]
But GPL protects the code from getting involved in closed source projects, so it keeps all versions and further development free for everyone.

**Reply by DoctorFuchs** [reply]
Do you know a license, with a property to disallow closed projects, but allow FOSS?

**Comment by DoctorFuchs** [comment]
You may be right. Because this is not a module it's maybe better to keep the license ...
